_posts/2018-01-24-cockpit-160.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Cockpit 160
+author: pitti
+date: 2018-01-24 10:00
+tags: cockpit linux
+slug: cockpit-160
+category: release
+summary: Cockpit with Kubevirt, Software Updates, and AppStream improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 160.
+
+### Add kubevirt Virtual Machines overview
+
+[kubevirt](http://www.kubevirt.io) is a project for running KVM virtual
+machines in [Kubernetes](https://kubernetes.io/). Cockpit's "Cluster"
+(Kubernetes/OpenShift) dashboard can now show the status of these VMs in the
+new "Virtual Machines" menu entry. This only appears when kubevirt is installed
+and active:
+
+![Cluster kubevirt list](/images/kubernetes-kubevirt-list.png)
+
+In the future this will be extended to also do operations on the VMs.
+
+Thanks to Jakub Niedermertl for this feature!
+
+### Redesign package list on Software Updates page and show RHEL Errata
+
+The table of available updates now looks and behaves much more consistently to
+other products that handle software packages, like
+[Satellite](https://www.redhat.com/de/technologies/management/satellite) and
+[Welder](https://github.com/weldr/welder-web):
+
+ * It now only shows the first line of the description and number of fixed
+   security issues or bugs in the table, and moves the full description and
+   detailled bug/CVE lists into an expander.
+
+ * It shows icons for the severity of the update, i. e. "security", "bug fix",
+   or "enhancement".
+
+ * On Red Hat operating systems it also shows the
+   [classification](https://access.redhat.com/security/updates/classification)
+   of security updates and links to the corresponding Errata.
+
+![Software Updates redesign and Errata](/images/packagekit-errata.png)
+
+ * [PackageKit](http://packagekit.org) often provides the update details in
+   [Markdown](https://en.wikipedia.org/wiki/Markdown) format. This now gets
+   rendered properly instead of shown verbatim in text.
+
+![Software Updates markdown](/images/packagekit-markdown.png)
+
+### AppStream handling on Apps page
+
+On Fedora, the Apps page now installs the `appstream-data` package when
+checking for new applications, which ships the
+[AppStream](https://www.freedesktop.org/wiki/Distributions/AppStream/) metadata
+for discovering new Cockpit applications.
+
+This Cockpit version also adds AppStream metadata for its own
+`cockpit-sosreport` package (which is not installed by default), so that it
+appears on the Apps page as an available extension.
```

Comment:
Should we mention that it doesn't appear just yet, but needs a new appstream-data as well?

Reply:
Yes, thanks for clarifying on IRC. Updated.